src/game/boe.main.cpp

```diff
@@ -91,6 +101,10 @@ effect_pat_type current_pat;
 short missile_firer,current_monst_tactic;
 short store_current_pc = 0;
 
+/* stuff that should not be here XXX */
```

Comment:
Again, `TODO:` or `FIXME:` prefix instead of a `XXX` suffix.

Reply:
done